hier_config/base.py

```diff
@@ -269,7 +238,7 @@ def unified_diff(self, target: Union[HConfig, HConfigChild]) -> Iterator[str]:
         # if a self child is missing from the target "- self_child.text"
         for self_child in self.children:
             self_iter = iter((f"{self_child.indentation}{self_child.text}",))
-            if target_child := target.children_dict.get(self_child.text, None):
+            if target_child := target.children.get(self_child.text, None):
```

Comment:
Help me understand this. You're using the walrus instead of doing:

```python
target_child = target.children.get(self_child.text, None)
if target_child:
   ...
```

Right?